packages/browser/src/plugins/in-app-plugin/index.ts

```diff
@@ -47,36 +49,65 @@ export function InAppPlugin(
         Gist.events.on('messageShown', (message: any) => {
             const deliveryId:string = message?.properties?.gist?.campaignId;
             if (typeof deliveryId != 'undefined' && deliveryId != '') {
-                _analytics.track(SemanticEvents.JourneyMetric, {
+                _analytics.track(JourneysEvents.Metric, {
                     'deliveryId': deliveryId,
-                    'metric': SemanticEvents.Opened,
+                    'metric': JourneysEvents.Opened,
+                });
+                return;
+            }
+            const broadcastId:Number = message?.properties?.gist?.broadcast?.broadcastIdInt;
+            if (typeof broadcastId != 'undefined' && broadcastId != 0) {
```

Comment:
Could be simplified to because the type is Number i think you just need to check for a truthy value
```suggestion
            if (broadcastId) {
```